lib/frosting/repository.rb

```diff
@@ -1,18 +1,45 @@
 require "active_support/core_ext/string/inflections"
+require "active_support/core_ext/module/delegation"
 
 module Frosting
   class Repository
+    def self.infer_presenter(resource)
+      "Presenters::#{resource.class.name}".constantize
+    end
```

Comment:
Can probs add `private_class_method :infer_presenter` under here, and same for `:procify`. Can also move them down under the public methods.